flax/nnx/nn/attention.py

```diff
@@ -401,13 +415,17 @@ def __init__(
         dtype=self.dtype,
         param_dtype=self.param_dtype,
         rngs=rngs,
+        # TODO: Add scale_metadata
+        # scale_metadata=ln_scale_metadata,
```

Comment:
@cgarciae do we want to add a new arg in MHA to propagate LayerNorm scale metadata for query and key? Should we have a single arg or 2 or zero?

Reply:
yes, we should have a metadata argument per Variable.